src/components/expanded-state/ChartExpandedState.js

```diff
@@ -1,5 +1,7 @@
 import { find } from 'lodash';
 import React, { useRef } from 'react';
+import { StatusBar } from 'react-native';
+import { isSamsungGalaxy } from '../../helpers/samsung';
```

Comment:
I'm a bit opposite of overusing this pattern. You know, there are thousands of devices and Samsungs are not very special in the group